tools/sklearn/utils.py

```diff
@@ -0,0 +1,242 @@
+import sys
+import os
+import pandas
+import re
+import pickle
+import numpy as np
+import xgboost
+import scipy
+import sklearn
+import ast
+from asteval import Interpreter, make_symbol_table
+from sklearn import metrics, model_selection, ensemble, svm, linear_model, naive_bayes, tree, neighbors
+
+
+def read_columns(f, c=None, c_option='by_index_number', return_df=False, **args):
+  data = pandas.read_csv(f, **args)
```

Comment:
If we start using real python files can you stick to 4 spaces indentation please.